lib/gui/app.js

```diff
@@ -296,3 +297,12 @@ app.controller('HeaderController', function(SelectionStateModel, OSOpenExternalS
   };
 
 });
+
+app.controller('StateController', function($state) {
+
+  /**
+   * @param {string} state - state page
+   */
+  this.is = $state.is;
```

Comment:
Can we make use of the `angular-if-state` directives (https://www.npmjs.com/package/angular-if-state) that we already have to replace this controller?